src/client/api.ts

```diff
@@ -445,8 +430,8 @@ export const createDeal = multiAsync(
 
 		const program = newCredixProgram(connection, wallet);
 
-		const principalAmount = new BN(toProgramAmount(principal));
-		const financingFeeAmount = new BN(toProgramPercentage(financingFee));
+		const principalAmount = new BN(principal.toNumber());
+		const financingFeeAmount: Ratio = { numerator: financingFee, denominator: 100 };
```

Comment:
What if the `financingFee` variable is 10.4?
Or are we not expecting those values

Reply:
yes that should be possible